zeebe/engine/src/main/java/io/camunda/zeebe/engine/state/EventApplier.java

```diff
@@ -54,7 +55,7 @@ public NoApplierForVersion(
           final Intent intent, final int recordVersion, final int latestVersion) {
         super(
             String.format(
-                "Expected to find an event applier for intent '%s' and version '%d', but '%s' is the latest supported version.",
+                "Expected to find an event applier for intent '%s' and version '%d', but '%s' is the latest supported version. This may happen during a rolling update..",
```

Comment:
🙃 ominous ellipsis intended or just a typo?

Reply:
rolling update............................. wait for the end!!!!! :smile: